saskatoon/fixtures/saskatoon.json

```diff
@@ -1 +1 @@
"Échelle"}}, {"model": "harvest.equipmenttype", "pk": 4, "fields": {"name": "Pochette des documents logistiques"}}, {"model": "harvest.equipmenttype", "pk": 5, "fields": {"name": "Balance"}}, {"model": "harvest.equipmenttype", "pk": 6, "fields": {"name": "Trousse de premiers soins"}}, {"model": "harvest.equipmenttype", "pk": 7, "fields": {"name": "Cueille-fruits"}}, {"model": "harvest.equipmenttype", "pk": 8, "fields": {"name": "Étuis à crayon et outils"}}, {"model": "harvest.equipmenttype", "pk": 9, "fields": {"name": "Hitch à vélo"}}, {"model": "harvest.equipmenttype", "pk": 10, "fields": {"name": "Caisses en bois"}}, {"model": "harvest.equipmenttype", "pk": 11, "fields": {"name": "Ciseaux"}}, {"model": "harvest.equipmenttype", "pk": 12, "fields": {"name": "Gants"}}, {"model": "harvest.equipmenttype", "pk": 13, "fields": {"name": "Bacs en plastique"}}, {"model": "harvest.property", "pk": 3, "fields": {"is_active": true, "authorized": true, "pending": true, "pending_contact_name": "", "pending_contact_phone": "", "pending_contact_email": null, "pending_newsletter": false, "pending_recurring": false, "geom": null, "owner": 1, "trees_location": "Backyard", "trees_accessibility": "Passer par l'arrière", "avg_nb_required_pickers": 2, "public_access": false, "neighbor_access": false, "compost_bin": true, "ladder_available": true, "ladder_available_for_outside_picks": false, "harvest_every_year": false, "number_of_trees": 2, "approximative_maturity_date": "2021-06-06", "fruits_height": 2, "street_number": "123", "street": "av. Mont-Royal", "complement": "apt. 4", "postal_code": "H2T 1V8", "publishable_location": "Mont-Royal, coin Buillon", "neighborhood": 19, "city": 1, "state": 1, "country": 1, "longitude": null, "latitude": null, "additional_info": "Je peux aider pour la ceuillète", "changed_by": null, "trees": [9, 14]}}, {"model": "harvest.harvest", "pk": 1, "fields": {"status": "Succeeded", "property": 3, "owner_present": true, "owner_help": true, "owner_fruit": true, "pick_leader": 2, "start_date": "2021-06-06T10:00:00Z", "end_date": "2021-06-06T10:00:00Z", "publication_date": "2021-05-27T18:43:51.899Z", "creation_date": "2021-05-26T21:42:50.701Z", "nb_required_pickers": 3, "about": "<p>Un tas de belles pommes et de noix de Grenoble au coin Mont-Royal et Buillon</p>", "changed_by": null, "trees": [9, 14], "equipment_reserved": [1]}}, {"model": "harvest.harvestyield", "pk": 1, "fields": {"harvest": 1, "tree": 9, "total_in_lb": 3.0, "recipient": 1}}, {"model": "harvest.harvestyield", "pk": 2, "fields": {"harvest": 1, "tree": 14, "total_in_lb": 6.0, "recipient": 2}}, {"model": "harvest.harvestyield", "pk": 3, "fields": {"harvest": 1, "tree": 9, "total_in_lb": 4.0, "recipient": 3}}, {"model": "harvest.harvestyield", "pk": 4, "fields": {"harvest": 1, "tree": 14, "total_in_lb": 2.0, "recipient": 1}}, {"model": "harvest.equipment", "pk": 1, "fields": {"type": 3, "description": "10ft aluminim step ladder", "owner": null, "property": 3, "shared": false}}]
```

Comment:
I don't think this file needs to re-generated, right?

Reply:
I have manually updated this file to use the changed `pending_contact_first_name` field and the added `pending_contact_family_name` field. I'm don't know if this is auto generated but I have updated according to the changed models. Let me know if there's any insight about this file that I'm missing out.

Reply:
@farhanmasud, I believe this file is auto-generated and the appropriate changes will be applied by the fixtures.
But @2ynn can probably speak more about this.

Reply:
@tristanlatr thanks for the info. I'll make the changes according to feedback of @2ynn

Reply:
If you want to regenerate it you'd need to first load it from a fresh database (on `develop` branch), then checkout your branch, edit the Nouveau Prorio first and last pending names from admin panel, then reexport the .json. So editing it manually for a couple changes is definitely ok. We could discuss if we really need this fixture. Note that the `saskatoon/fixtures/init` uses the individual, human readable .json files